addon/mixins/view-support.js

```diff
@@ -22,18 +22,14 @@ export default Ember.Mixin.create({
     Ember.run.scheduleOnce('afterRender', this, this._scrollToElemPosition);
   },
 
-  didInsertElement() {
-    this._super(...arguments);
-    this._scrollToElemPosition();
-  },
-
-  _scrollToElemPosition() {
+  _scrollToElemPosition: on('didInsertElement', function() {
```

Comment:
I believe it's just personal preference to avoid the need of calling `this._super`, but there's no change in behavior whatsoever. Just let me know if you prefer to keep the former and I'll revert the change. :ok_hand: